src/collective/behavior/amp/browser/view.pt

```diff
@@ -217,6 +218,8 @@
         tal:content="structure sticky_ad">
     </amp-sticky-ad>
 
+    <amp-pixel tal:condition="pixel" tal:replace="structure pixel" />
```

Comment:
If you follow my suggestion to change from simple adapter to view then maybe you could remove the `pixel` property from the view and just write 

`<amp-pixel tal:replace="structure context/@@amp-tracking-pixel" />`

Again, write a default `@@amp-tracking-pixel` to avoid a `tal:condition`.